pennylane_lightning/src/Util.hpp

```diff
@@ -17,10 +17,38 @@
  */
 #pragma once
 
+#include <memory>
```

Comment:
What's `<memory>` for?  I see `unique_ptr` is in the memory std, but elsewhere its getting imported with `using std::unique_ptr;`.

Reply:
That's the header itself, required for `unique_ptr`. `using std::unique_ptr;` introduces `unique_ptr` from the `std` namespace.